man/systemd-activate.xml

```diff
@@ -104,6 +105,13 @@
       </varlistentry>
 
       <varlistentry>
+        <term><option>-d</option></term>
+        <term><option>--datagram</option></term>
+
+        <listitem><para>Listen to datagram sockets.</para></listitem>
```

Comment:
Isn't that self  explanatory  If no option is given then it would default to stream else user will use -d option to use Datagram ?